README.md

```diff
@@ -37,10 +37,28 @@ provide the required tools), you can disable the `deb` and `rpm` Maven profiles
 the command line:
 
 ```bash
-mvn clean package -P \!deb,\!rpm
+mvn clean package -P \!deb,\!rpm,\!tarball
```

Comment:
this command will run also the `test` profile which requires docker, should we also append `,\!test`, to skip them?

Reply:
It's OK, as long as is only for client's local building.